webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,14 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $("textarea.js-autoExpand").on("input", function() {
+     $(this).css({"height":0});
```

Comment:
This is looking better!

(Small tip, when using jQuery, if you're just setting a single property via `$.fn.css`, convention is to just do `$(foo).css("prop", val)`, rather than use the object syntax.)

That said, I think we want to do something a bit nicer than using 0. What if we set a local variable to the initial height of the textarea, then use that to "reset" the height. That way it will never get shorter than it initially was:


```
this.autoExpand = function() {
  var initialHeight = $("textarea.js-autoExpand").height();
  $("textarea.js-autoExpand").on("input", function() {
  $(this).css({"height": intialHeight});
  ...